src/Components/Web.JS/src/Services/NavigationUtils.ts

```diff
@@ -112,16 +112,16 @@ function findAnchorTarget(event: MouseEvent): HTMLAnchorElement | null {
     // legacy browsers that don't support it by falling back on the older logic, even
     // though it won't work properly with ShadowDOM. This can be removed in the next
     // major release.
-    return findClosestAnchorAncestorLegacy(event.target as Element | null, 'A');
+    return findClosestAnchorAncestorLegacy(event.target as Element | null);
   }
 }
 
-function findClosestAnchorAncestorLegacy(element: Element | null, tagName: string) {
+function findClosestAnchorAncestorLegacy(element: Element | null) {
```

Comment:
I think we should just remove `findClosestAnchorAncestorLegacy` for 9.0. It's not required for any of our supported browsers now, as far as I know.

Reply:
That would also mean removing the `_blazorDisableComposedPath` support above.

Reply:
Done